### ISSUE TITLE
Heroes of the ruin - Hangs at title screen

### ISSUE THREAD

**Issue by JohnGodgames** [citra-emu/citra#1157]
![heroes of the ruin](https://cloud.githubusercontent.com/assets/11453507/9866752/b2620ad8-5b66-11e5-9cde-47634950755d.PNG)

LOG:
http://pastie.org/10420479

**Comment by mailwl** [comment]
easy to fix, just implement AC:U function "IsConnected", for example
```
static void IsConnected(Service::Interface* self) {
    u32* cmd_buff = Kernel::GetCommandBuffer();

    cmd_buff[1] = 0; // No error
    cmd_buff[2] = 0; // Connection type set to none

    LOG_WARNING(Service_AC, "(STUBBED) called");
}
```

**Comment by woody2014** [comment]
could you submit this as a pull request pls

**Comment by JohnGodgames** [comment]
Fixed by #1429
